lib/tasks/demo-build.js

```diff
@@ -353,17 +353,6 @@ module.exports = function (cfg) {
 				}
 			}
 
-			if (demos.length === 0 && !config.suppressErrors) {
-				let noDemosError = 'No demos were found';
-				if (config && demoFilters) {
-					noDemosError += ' for ' + String(demoFilters);
-				}
-				noDemosError += '.';
-
-				const e = new Error(noDemosError);
-				e.stack = '';
-				throw e;
-			}
 			// Check if obt is watching for Sass changes.
```

Comment:
does it just silently do nothing if the person has asked to run demos and they don't exist?

Reply:
Only if the --demo-filter flag is also passed
```
❯ ../../../origami-build-tools/lib/origami-build-tools-cli.js demo
 ✖ Compiling Demos   
    → Sass file not found: /Users/jake.champion/Code/repo-data-cli/components/o-table… 
❯ ../../../origami-build-tools/lib/origami-build-tools-cli.js demo --demo-filter=jake
✔ Compiling Demo
```

Reply:
hm, i wonder if it would be nice to print something like "skipped: no demos matched the given filter"?

Reply:
Yes, I like that idea a lot!

Reply:
Done!